frontend/public/style/_vars.scss

```diff
@@ -23,6 +23,8 @@ $font-family-base: var(--pf-global--FontFamily--sans-serif);
 $font-path: '~patternfly/dist/fonts/';
 // Patternfly 3 defaults to 12px, which is too small in many cases.
 $font-size-base: 14px;
+// Patternfly 4 defaults to 16px
+$font-size-pf4-base: ($font-size-base + 2);
```

Comment:
I would just say `16px`. This var is independent of `$font-size-base`

Reply:
Can we make this a var just for the masthead icon font size? Is that the only place this would be used?

Reply:
switched to `$pf-header-icon-fontsize`